src/components/Congress/Motions/index.js

```diff
@@ -15,13 +15,18 @@ export default function Motions() {
     dispatch(congressActions.getMotions.call());
   }, [dispatch]);
 
-  if (!motions || motions.length < 1) {
+  useEffect(() => {
+    if (!motions?.names) return;
+    dispatch(identityActions.getIdentityMotions.call(motions.names));
```

Comment:
why do it as useeffect instead of doing it in the call or as a param when fetching the data, especially given this appears in multiple places?